sdk/src/main/java/com/podio/sdk/Podio.java

```diff
@@ -245,33 +249,37 @@ public static void setup(Context context, String clientId, String clientSecret)
      *         The pre-shared Podio client id.
      * @param clientSecret
      *         The corresponding Podio client secret.
-     * @param sslSocketFactory
+     * @param userAgent
+     *         Optional user agent.
+     * @param volleySslSocketFactory
      *         Optional custom SSL socket factory to use in the HTTP requests.
+     * @param androidAsyncHttpSslSocketFactory
+     *         Optional custom SSL sockey factory to use for uploading files.
      */
-    public static void setup(Context context, String scheme, String authority, String clientId, String clientSecret, SSLSocketFactory sslSocketFactory) {
-        restClient.setup(context, scheme, authority, clientId, clientSecret, sslSocketFactory);
-
+    public static void setup(Context context, String scheme, String authority, String clientId, String clientSecret, String userAgent, SSLSocketFactory volleySslSocketFactory, cz.msebera.android.httpclient.conn.ssl.SSLSocketFactory androidAsyncHttpSslSocketFactory) {
```

Comment:
What do you mean? It is reflected: https://github.com/podio/android-app/blob/fileupload/app/src/main/java/com/podio/sdk/SuperPodio.java#L128